# Introdução à Ciência de Dados - UFPB
Professor: Yuri Malheiros

Aluna: Aurenívia Gomes da Costa
## Medidas de Centralidade e Dispersão

A partir do dataset `movies.csv` calcule a média, mediana e desvio padrão para as colunas:

In [3]:
import pandas as pd

movies = pd.read_csv("movies.csv")
movies

Title    US Gross  ...  IMDB Rating  IMDB Votes
0                 The Land Girls    146083.0  ...          6.1      1071.0
1         First Love, Last Rites     10876.0  ...          6.9       207.0
2     I Married a Strange Person    203134.0  ...          6.8       865.0
3           Let's Talk About Sex    373615.0  ...          NaN         NaN
4                           Slam   1009819.0  ...          3.4       165.0
...                          ...         ...  ...          ...         ...
3196  Zack and Miri Make a Porno  31452765.0  ...          7.0     55687.0
3197                      Zodiac  33080084.0  ...          NaN         NaN
3198                        Zoom  11989328.0  ...          3.4      7424.0
3199         The Legend of Zorro  45575336.0  ...          5.7     21161.0
3200           The Mask of Zorro  93828745.0  ...          6.7      4789.0

[3201 rows x 16 columns]

### US Gross

In [6]:
us_gross = movies['US Gross']
print(f"US GROSS - Média: {us_gross.mean()}, Mediana: {us_gross.median()}, Desvio Padrão {us_gross.std()}")

US GROSS - Média: 44002085.163744524, Mediana: 22019465.5, Desvio Padrão 62555311.39066204


### Production Budget

In [7]:
coluna = movies['Production Budget']
print(f"PRODUCTION BUDGET - Média: {coluna.mean()}, Mediana: {coluna.median()}, Desvio Padrão {coluna.std()}")

PRODUCTION BUDGET - Média: 31069171.4484375, Mediana: 20000000.0, Desvio Padrão 35585913.44464446


### Running Time

In [9]:
coluna = movies['Running Time (min)']
print(f"RUNNING TIME - Média: {coluna.mean()}, Mediana: {coluna.median()}, Desvio Padrão {coluna.std()}")

RUNNING TIME - Média: 110.19354838709677, Mediana: 107.0, Desvio Padrão 20.171013677535424


### IMDB Rating

In [10]:
coluna = movies['IMDB Rating']
print(f"IMDB Rating - Média: {coluna.mean()}, Mediana: {coluna.median()}, Desvio Padrão {coluna.std()}")

IMDB Rating - Média: 6.283467202141896, Mediana: 6.4, Desvio Padrão 1.2522899386004838


### IMDB Votes

In [11]:
coluna = movies['IMDB Votes']
print(f"IMDB Votes - Média: {coluna.mean()}, Mediana: {coluna.median()}, Desvio Padrão {coluna.std()}")

IMDB Votes - Média: 29908.64457831325, Mediana: 15106.0, Desvio Padrão 44937.58233504965


### Análise dos Resultados

Quais colunas tiveram valores muito diferentes entre média e mediana? Quais os seus desvios padrão?

1.  US GROSS - 62555311.39066204
2.  Production Budget - 35585913.44464446
3.  IMDB Votes - 44937.58233504965





Que conclusões podemos tirar desses resultados?

Podemos concluir que esses dados estão espalhados em uma ampla gama de valores, e os dados são bastante heterogêneos